vdom/core.py

```diff
@@ -8,6 +8,7 @@
 """
 
 from jsonschema import validate, Draft4Validator, ValidationError
+from functools import partial
```

Comment:
where are you actually using `partial` here? I don't see it…

Reply:
Good catch! I was using it earlier in create_component, but removed it in favor of an inner function. Removed import now.